scripts/README.md

```diff
@@ -0,0 +1,14 @@
+### Instructions to generate csv from i2b2 Obesity Challenge xml files
+
+First gain access to dataset. Once you have the access, you need to download the following xml files and rename exactly as below:
```

Comment:
I am thinking that it might be better to read the files directly from the URL instead of requiring this manual step. We can include the access variables using environment variables to not expose the data. What do you think?

Reply:
Yes, sounds good. I am having a little problems implementing this. Can you guide me through this ?

Reply:
Yeah, absolutely. What is the issue?